controllers/dateRangeController.js

```diff
@@ -55,7 +55,7 @@ export const handleDateRange = async (req, res) => {
     res.status(500).send('Error processing file.');
   } finally {
     console.log('Initiating cleanup...');
-    const filesToClean = [inputFile, outputFile, svgFilePath].filter(Boolean);
-    await cleanupFiles(filesToClean, svgDirPath);
+    // const filesToClean = [inputFile, outputFile, svgFilePath].filter(Boolean);
+    // await cleanupFiles(filesToClean, svgDirPath);
   }
```

Comment:
This should be retained. Once the SVG is sent to the client, a call to `cleanupFiles` removes all the files piled up on the server.